### DISCUSSION TITLE
GatorDesk Development Discussions!

### DISCUSSION THREAD

**Discussion by BHAVANA1497** [BHAVANA1497/GatorDesk#15] [Announcements]
sprint-1

**Comment by Nitya-vaishnavi** [comment]
/createAnnouncement API created. @akhil-maddukuri Have you tested with all the test cases including the border-line cases?

**Reply by akhil-maddukuri** [reply]
Yes, the test cases are being correctly handled by the program. We can close any existing issues regarding this.

**Reply by Nitya-vaishnavi** [reply]
Okay, code changes pushed

**Comment by HarshithaMyadam** [comment]
Hey @BHAVANA1497 , I have raised a pull request for the updated landing page in addition to the Announcement component. Could you review the code and proceed accordingly?

**Reply by BHAVANA1497** [reply]
Hey @HarshithaMyadam, I reviewed the Pull Request and merged it to the main branch. Thanks

**Comment by HarshithaMyadam** [comment]
@BHAVANA1497 I have noticed that the carousel feature is not working as expected. Please create a new issue for this bug if needed.

**Reply by BHAVANA1497** [reply]
Thanks for letting me know, I raised an issue and I am working on it. Once I fix the issue, I will raise a PR for you to approve.

**Comment by BHAVANA1497** [comment]
Heyy @HarshithaMyadam, Let's change the theme of CSS to teal and modify the header and footer accordingly. what do you think?

**Reply by HarshithaMyadam** [reply]
Sure @BHAVANA1497. Let us work on it together and make the web page aesthetically pleasing.
